op-node/rollup/derive/frame.go

```diff
@@ -8,10 +8,10 @@ import (
 	"io"
 )
 
-// Frames cannot be larger than 1 MB.
 // Data transactions that carry frames are generally not larger than 128 KB due to L1 network conditions,
 // but we leave space to grow larger anyway (gas limit allows for more data).
-const MaxFrameLen = 1_000_000
+// For AltDA, frames size can be larger. Setting to 16 MB as current blob limit for EigenDA.
+const MaxFrameLen = 16_000_000
```

Comment:
How is the final size decided? Does that happen in the node or in the batcher?

Reply:
This change applies to derivation pipeline, and affects only to the limit enforced to op-node. op-batcher has a cmdflag to set the batch size limit (`--max-l1-tx-size-bytes`) and this is not affected by `MaxFrameLen`.

16 MB was chosen as it is currently the limit of blob size on EigenDA Holesky.